### ISSUE TITLE
zoekt-webserver: Work With Empty Index Directory

### ISSUE THREAD

**Issue by dharesign** [google/zoekt#130]
I have configured two services, one running `zoekt-indexserver` and another `zoekt-webserver`. On a freshly configured system, the web server fails to start until the index server has written the first item to the index directory.

It fails [here](https://github.com/google/zoekt/blob/1b04f75645d067232d26f6e81fbd533613efa28e/shards/watcher.go#L84).

It would be nice if the web server could come up immediately.

**Comment by hanwen** [comment]
yeah, I added this to guard against operator error, but maybe we should just remove this check. Send a change? (see CONTRIBUTING)
